### DISCUSSION TITLE
low-res test, high-res final render

### DISCUSSION THREAD

**Discussion by tomByrer** [remotion-dev/remotion#81] [Q&A]
I'm just used to video editing suites that have low resolution when editing, then highest res when cutting the master.

What's the best workflow for this with remotion please?  (Noting jpeg/mp4 has just been added.)

**Comment by JonnyBurger** [comment]
Since it's just the DOM, there isn't something analogue to reducing the resolution to only render every nth pixel I think.
But you can reduce the preview size with the dropdown (25% / 50% / 100% / Fit) and if it's smaller maybe it uses less computing power?

I am not so sure if this is effective 🤔

**Comment by monophthongal** [comment]
If you want to be able to render a low-res MP4 preview (instead of just using the preview UI) one solution would be to add another `<Composition>` in your project that uses the same component as your main Composition, but with lower values for the FPS, width and height props. Then render that composition for previews.

The tricky part would be making sure your components are responsive and work the same in both Compositions, but that should be possible with CSS and `useVideoConfig()`.
